### DISCUSSION TITLE
Question aabout KFoldDataset and kfold-cross-valid.py

### DISCUSSION THREAD

**Discussion by BingliangLi** [open-mmlab/mmclassification#1179] [Q&A]
Hi, I would like to use 5 fold cross-validation to train my model, I found there are two possible implantations:

- Use `KFoldDataset`, and manually repeat 5 times
- Use `kfold-cross-valid.py`, only need to run it once

But I'm not sure if `kfold-cross-valid.py` support multi-GPU training, I can't find any related documentation, and by the script's content, I think it probably does not. I want to ask if I want to perform a 5-fold cross-validation with multiple GPUs, is using KFoldDataset the only way?

**Comment by mzr1996** [comment]
Hello, which version are you using? 1.x or master?
I will try to test the multi-gpu training for k-fold validation recently

**Reply by BingliangLi** [reply]
Hi, I'm using the lastest version.

**Reply by mzr1996** [reply]
Ok, welcome to use our 1.x branch, which will become the master branch at the end of this year. And of course I will also test it on the master branch.
